Template/Trigger/ElementVisibilityTrigger.web.js

```diff
@@ -235,9 +230,82 @@
             };
         }
 
+        function getSelectors() {
```

Comment:
fyi in line 122-197 maybe we can now as well use `TagManager.dom.bySelector(selectors).`?`

Reply:
Refactored the code